src/stories/Seekers/CanAddSeeker/endpoint.spec.js

```diff
@@ -111,11 +114,14 @@ describe("Test API Seekers/CanAddSeeker", () => {
         company_type: "start-up",
         company_employee_strenget: "50-100",
         current_salary: {
-          currency: {},
-          range: {},
+          currency: "INR",
+          range: {
+            from: "500000",
+            to: "700000",
+          },
         },
         note_for_recruiter: "",
-        notice_period: "10-20 days",
+        notice_period: "10 days",
```

Comment:
same for notice period, it should always be in days.
2 months => 60
3 months => 90

key: notice_period_in_days